src/components/author-template.js

```diff
@@ -14,13 +14,17 @@ export default function AuthorTemplate(props) {
     <Layout>
       <Helmet>
         <title>{name}</title>
+        <meta
+          name="description"
+          content={`More posts by ${name} | ${props.data.wp.generalSettings.description}`}
```

Comment:
maybe not relevant for SEO, description for twitter and FB are different than this one (don't have the name part)

Reply:
Ah nice catch, my bad! This is fixed up.